pallets/proposals/src/lib.rs

```diff
@@ -562,6 +565,7 @@ pub struct Project<T: Config> {
     pub cancelled: bool,
     pub funding_type: FundingType,
     pub deposit_id: DepositIdOf<T>,
+    pub payment_address: [u8; 20],
```

Comment:
This is meant to signal where the tokens will end up. check telegram when you get a minute!

Reply:
Sure